static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js

```diff
@@ -66,12 +66,10 @@ define([
         });
     }
 
-    function isMtRecTest() {
-        var MtRec1Test = ab.getParticipations().MtRec1,
-            MtRec2Test = ab.getParticipations().MtRec2;
+    function isViewabilityTest() {
```

Comment:
Um and here

Is there a reason why this code is copy-pasted? It'd be better if you put it into its own module so it's only included once in the outputted JavaScript. I know it's basically gzipped out of the download but it does still have to be parsed and represented in memory as separate functions.